File: NbaTracker.Api/NbaTracker.Common/DataContracts/PlayerBoxScore.cs
namespace NbaTracker.Common.DataContracts;

public class PlayerBoxScore
{
    public int PersonId { get; set; }
    public required string FirstName { get; set; }
    public required string LastName { get; set; }
    public required string JerseyNumber { get; set; }
    public bool Starter { get; set; }
    public bool Played { get; set; }
    public bool OnCourt { get; set; }
    public int SortOrder { get; set; }
    public PlayerGameStats GameStats { get; set; } = new PlayerGameStats();
}

File: nba-tracker-app/src/app/DataContracts/PlayerBoxScore.ts
import {PlayerGameStats} from "./PlayerGameStats";

export interface PlayerBoxScore {
  personId: number;
  firstName: string;
  lastName: string;
  jerseyNumber: string;
  starter: boolean;
  played: boolean;
  onCourt: boolean;
  SortOrder: number;
  gameStats: PlayerGameStats;
}
